_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+  - In some special cases, the reviewer may wish to disclose his/her identity to the manuscript authors, the editor should be consulted. There may be instances where the editor requests that the reviewer retains anonymity. The decision to disclose such information to authors is the sole discretion of the editor. However, such decisions should be accompanied by appropriate rationale from the editor. 
+  - Reviewers may disclose publicly that they have served as a reviewer for LiveCoMS. However, under no circumstances should the reviewer identify himself/herself as the reviewer of a specific manuscript. 
+- Reviewers should explain and support their judgments adequately so that editors and authors may understand the basis of their comments. Any statement that an observation, derivation, or argument had been previously reported should be accompanied by the relevant citation. Unsupported assertions by reviewers (or by authors in rebuttal) are of little value and should be avoided.
+- A reviewer should be alert to failure of authors to cite relevant work by other scientists, bearing in mind that complaints that the reviewer’s own research was insufficiently cited may seem self-serving. A reviewer should call to the editor’s attention any substantial similarity between the manuscript under consideration and any published paper or any manuscript submitted concurrently to another journal.
+- A reviewer should act promptly, submitting a report in a timely manner. Should a reviewer receive a manuscript at a time when circumstances preclude prompt attention to it, the unreviewed manuscript should be returned immediately to the editor. Alternatively, the reviewer might notify the editor of probable delays and propose a revised review date.
+- Reviewers should not use or disclose unpublished information, arguments, or interpretations contained in a manuscript under consideration, except with the consent of the author. If this information indicates that some of the reviewer’s work is unlikely to be profitable, the reviewer, however, could ethically discontinue the work. In some cases, it may be appropriate for the reviewer to write the author, with copy to the editor, about the reviewer’s research and plans in that area.
+- The review of a submitted manuscript may sometimes justify criticism, even severe criticism, from a reviewer. However, in no case is personal criticism of the author considered to be appropriate. 
+- Reviewers should notify editors of concerns with respect to manuscripts that report research that, based on current understanding, can be reasonably expected to provide knowledge, products, or technologies that could be directly misapplied by others to pose a threat to public health and safety, agricultural crops and other plants, animals, the environment, or materiel. 
```

Comment:
I don't think this last one applies to us either?

Reply:
Yeah, since it is all public, I think so.  It's also unlikely that there is any such research (since most of the material is review), but might as well.